packages/shared/src/multisig.proxy.ts

```diff
@@ -239,7 +241,8 @@ async function multisigWithPureProxyTest<
 
   await checkEvents(finalApprovalEvents, 'multisig')
     .redact({
-      redactKeys: /height/,
+      // The approving address is the proxy, which is not deterministic.
+      redactKeys: /approving|multisig/,
```

Comment:
@andreitrand regarding our conversation; while multisig addresses are indeed deterministic,
1. a multisig account that has a proxy as signatory, or
2. a proxy account created from a multisig account
are not; this was causing some false positive test failures due to this `multisig` address field differing in snapshots.

I overlooked this in my review of #429 , apologies.